webcompat/static/config_data/labels.json

```diff
@@ -0,0 +1,182 @@
+[
+  {
+    "url": "https://api.github.com/repos/webcompat/web-bugs/labels/status-duplicate",
+    "name": "status-duplicate",
+    "color": "cccccc"
```

Comment:
> Another option is instead of building the file, we just download it from GitHub from time to time? Maybe every 24 hours? (labels don't get added that frequently...)

Or do something cool like set up a webhook handler for a "new label was created" event (should such a thing exist).